wordpress-mobile/gutenberg-mobile
# Jetpack Tiled Gallery: Inner blocks implementation

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/4000

Related:
- https://github.com/Automattic/jetpack/pull/21135

### To test

1. In the terminal, run `npm run start:reset` 
2. In another terminal window, run `npm run core ios`
3. Add the Tiled Gallery block
4. Verify that you can add, remove, re-arrange images and that each is an inner block
5. Repeat the above steps for Android, running `npm run core android` in Step 2

PR submission checklist:
